roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -77,27 +77,22 @@ server {
 
   ssl_buffer_size 1400; # 1400 bytes to fit in one MTU
 
-  {% if item.value.ssl.provider | default('manual') != 'self-signed' -%}
+  {% if env != 'development' -%}
   add_header Strict-Transport-Security "max-age={{ [hsts_max_age, hsts_include_subdomains, hsts_preload] | reject('none') | join('; ') }}";
   {% endif -%}
 
-  {% if item.value.ssl.client_cert_url is defined -%}
-  ssl_verify_client       on;
-  ssl_client_certificate  {{ nginx_ssl_path }}/client-{{ (item.value.ssl.client_cert_url | hash('md5'))[:7] }}.crt;
+  {% if site_ssl.manual is defined and site_ssl.manual.client_cert_url is defined -%}
```

Comment:
Client cert can be used in non-manual setups as well.

Reply:
Removed the `site_ssl.manual is defined` part of the conditional 👍